models/repo.go

```diff
@@ -1668,8 +1668,13 @@ func TransferOwnership(doer *User, newOwnerName string, repo *Repository) error
 
 	if err = watchRepo(sess, doer.ID, repo.ID, true); err != nil {
 		return fmt.Errorf("watchRepo: %v", err)
-	} else if err = transferRepoAction(sess, doer, owner, repo); err != nil {
-		return fmt.Errorf("transferRepoAction: %v", err)
+	}
+
+	// Remove watch for organization.
+	if owner.IsOrganization() {
+		if err = watchRepo(sess, owner.ID, repo.ID, false); err != nil {
+			return fmt.Errorf("watchRepo [false]: %v", err)
```

Comment:
@lunny should we use `log.Error`  from `code.gitea.io/gitea/modules/log` in models/* too?

Reply:
Finally we should use a standard log library to do that. Maybe code.gitea.io/log . Here we just return the error.

Reply:
You can use modules/log in models - it's used in lots of places already.